Gemfile

```diff
@@ -128,3 +130,24 @@ group :development do
   gem 'stackprof'
   gem 'bundler-audit'
 end
+
+group :worker do
+  # These gems are only used in workers (and just so happen to slow down app startup
+  # by quite a bit), so we exclude them from all environments other than RAILS_ENV=worker.
+
+  # Document understanding
+  gem 'htmlentities'
+  gem 'birch', github: 'billthompson/birch', branch: 'birch-ruby22'
+
+  # Todo this is a super useful gem but no longer maintained and too full of bugs to use; we should find/write an alternative.
+  # gem 'treat', :git => 'https://github.com/indentlabs/treat-gem.git'
+
+  gem 'engtagger'
+  gem 'ibm_watson'
+
+  # These are unfortunately necessary as a hack to get treat running properly
+  gem 'rubyzip', '>= 1.0.0' # will load new rubyzip version
+  gem 'zip-zip' # will load compatibility for old rubyzip API.
+end
+gem 'engtagger'
```

Comment:
`engtagger` and `ibm_watson` should be isolated to `worker` environments if possible.